build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
```

Comment:
Unrelated, but I couldn't compile without this, it tried to include "com.android.support" by default and couldn't find it.